### PR TITLE
added AddChild

### PR DESCRIPTION
**Pull request by gbanares**
modified:
client/src/app.js
routes/api/children.js
client/src/components/AddChild/

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -56,7 +57,7 @@ class App extends Component {
         <Navbar updateUser={this.updateUser} loggedIn={this.state.loggedIn} />
         {/* greet user if logged in: */}
         {this.state.loggedIn &&
-          <p>Join the party, {this.state.username}!</p>
+          <p>Logged In as user: {this.state.username}</p>
         }
         {/* Routes to different components */}
```
Comment (DChan0319) [inline client/src/App.js]: Is this necessary comment? I'm sure the Routes tag speak for itself.

File: client/src/components/AddChild/addchild.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+class AddChild extends Component {
+  state = {
+    name: "",
+    budget: 0,
+  };
+
+  handleInputChange = event => {
+    const { name, value } = event.target;
+    this.setState({
+      [name]: value
```
Comment (DChan0319) [inline client/src/components/AddChild/addchild.js]: do you need brackets here?

File: client/src/components/AddChild/addchild.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  };
+
+  handleFormSubmit = event => {
+    event.preventDefault();
+    if (this.state.name && this.state.budget) {
+      API.getAddChild({
+        name: this.state.name,
+        budget: this.state.budget
+      })
+        // .then(res => this.loadJobs())
```
Comment (DChan0319) [inline client/src/components/AddChild/addchild.js]: why is this .then commented out?

File: client/src/components/AddChild/addchild.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+  render() {
+    return (
+      <Container fluid>
+        <Row>
+          <Col size="md-6">
+            <Jumbotron>
+              <h1>Create Child</h1>
+            </Jumbotron>
+            <form>
```
Comment (DChan0319) [inline client/src/components/AddChild/addchild.js]: You can probably extract out this form component to its own file and then import it in here, Your return for your render function is getting long, but the downside is having to pass down the props even more, but y'know it shouldn't be that bad.

File: routes/api/children.js [hunk 5]
```diff
@@ -1,7 +1,8 @@
 const childrenController = require('../../controllers/childrenController');
 
 module.exports = (app) =>{
-    app.post('/api/createChild',childrenController.createChild);
+    app.post('/api/AddChild',chilrenController.createChild);
+    // app.post('/api/createChild',childrenController.createChild);
```
Comment (DChan0319) [inline routes/api/children.js]: Is this route going to be use in the future? I assume so just throw on a TODO: or FIXME:

File: client/src/App.js (hunk 1 above)
Comment (gbanares) [reply client/src/App.js]: True.

Comment (gbanares) [reply client/src/App.js]: Removed.

File: routes/api/children.js (hunk 5 above)
Comment (gbanares) [reply routes/api/children.js]: Yes, we are planning for future use. Will add the TODO or FIXME

**Comment by gbanares** [comment]
I'm going to close this pull request and try another.

**Pull request closed without merging**